src/components/ChoiceList/tests/ChoiceList.test.tsx

```diff
@@ -103,6 +103,27 @@ describe('<ChoiceList />', () => {
         expect(choiceElements.contains(children)).toBe(true);
       });
 
+      it('renders a choice with children wrapper div when choice is selected', () => {
+        const selectedIndexes = [2];
+        const selected = selectedIndexes.map((index) => choices[index].value);
+
+        choices = [
+          choices[0],
+          choices[1],
+          {
+            ...choices[2],
+            renderChildren: renderChildrenSpy,
+          },
+        ] as any;
+
+        const choiceElements = shallowWithAppProvider(
+          <ChoiceList selected={selected} choices={choices} />,
+        );
+
+        expect(renderChildrenSpy).toHaveBeenCalled();
```

Comment:
It's recommended to declare the spy in the test itself rather than in the describe, otherwise you'll end up re-using the same spy throughout your test.

Reply:
Yeah agreed. Wondering why I didn't do it when I wrote the initial tests 🤔

Reply:
🤦‍♂️  Will address & fix others